server/pulp/server/managers/repo/group/cud.py

```diff
@@ -35,9 +34,8 @@ def create_repo_group(group_id, display_name=None, description=None, repo_ids=No
         """
         if repo_ids:
             # Check if ids in repo_ids belong to existing repositories
-            repo_query_manager = manager_factory.repo_query_manager()
             for repo_id in repo_ids:
-                repo_query_manager.get_repository(repo_id)
+                model.Repository.objects.get_repo_or_missing_resource(repo_id)
```

Comment:
Nice catch.

Rather than push the code and hope that you happen to notice it, I'll post my proposed change here and get feedback first.

``` python
# Check if ids in repo_ids belong to existing repositories
existing_repos = model.Repository.objects(repo_id__in=repo_ids)
if len(existing_repos) != len(repo_ids):
    existing_repo_ids = set([repo.repo_id for repo in existing_repos])
    non_existing_repo_ids = list(set(repo_ids) - existing_repo_ids)
    raise pulp_exceptions.MissingResource(repositories=non_existing_repo_ids)
```